kicad_jlc_tool/generate_jlc_files.py

```diff
@@ -23,8 +23,8 @@
 import logging
 import errno
 
-from jlc_lib.cpl_fix_rotations import ReadDB, FixRotations
-from jlc_lib.generate_bom import GenerateBOM
+from kicad_jlc_tool.jlc_lib.cpl_fix_rotations import ReadDB, FixRotations
```

Comment:
Can we stick to jlc_kicad_tools? Just so all the names match (repository name, library name, package name, etc).